calico-cloud/threat/web-application-firewall.mdx

```diff
@@ -52,6 +52,16 @@ You simply enable WAF in Manager UI, and determine the services that you want to
 - WAF is not supported for host-networked client pods
 - When selecting and deselecting traffic for WAF, active connections may be disrupted
 
+:::note warning
+
+Do not enable WAF for the system services with the following namespace prefixes as doing so may result in an undesired cluster state:
+
+- `tigera-*`
```

Comment:
Is the * meant as a wildcard, or is it part of the prefix?

Reply:
I think it is wildcard, confirming with dev.

Reply:
Not wild card.

Reply:
it's a prefix match, according to the frontend code.. so technically still a wildcard?